pkg/persistence/sqlite_schema.go

```diff
@@ -20,7 +20,7 @@ spec:
         constraints:
           notNull: true
       - name: updated_at
-        type: integer
+        type: timestamp
```

Comment:
these were failing to be converted to time.Time in go. sqlite had a workaround for this by using the `timestamp` column type which will handle it internall.

Reply:
@divolgin ...  thoughts on this?

Reply:
1st reference: https://github.com/mattn/go-sqlite3/issues/748#issuecomment-538643131

Reply:
2nd reference: https://github.com/mattn/go-sqlite3/issues/190#issuecomment-186704817